compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/JvmBuiltinOptimizationLowering.kt

```diff
@@ -35,17 +35,11 @@ class JvmBuiltinOptimizationLowering(val context: JvmBackendContext) : FileLower
 
     companion object {
         fun isNegation(expression: IrExpression, context: JvmBackendContext): Boolean {
```

Comment:
I think both these functions can now be inlined

Reply:
They can. Done.